src/libraries/Microsoft.PowerFx.Core/Parser/ParseResult.cs

```diff
@@ -72,5 +73,13 @@ internal ParseResult(TexlNode root, List<TexlError> errors, bool hasError, List<
             err.FormatCore(sb);            
             return $"Err#{++i} {sb}";
         }));
+
+        /// <summary>
+        /// Converts the current formula into an anonymized format.
+        /// </summary>
+        public string GetAnonymizedFormula()
```

Comment:
>GetAnonymizedFormula [](http://example.com/codeflow?start=22&length=20)

This is good. 
And as a method, we can add overloads with other parameters later if we want to customize.